style.css

```diff
@@ -87,3 +88,278 @@ body {
+  padding-left: 15px;
+  padding-right: 30 px;
+  font-size: 16px;
+  color: #091e42;
+}
+
+.list_programs {
+  display: flex;
+  align-content: space-between;
+  flex-direction: row;
+  padding: 10px 12px;
+}
+
+.boton1 {
+  height: 48px;
+  background: #ff6b00;
+  margin-left: 15px;
+  font-family: 'Inter', sans-serif;
+  font-size: 17px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  letter-spacing: 0.03em;
+  color: #fff;
+}
+
+.projects {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  background-image: url(./images/fondo.svg);
+  background-repeat: no-repeat;
+  width: 327px;
+}
+
+.tituloPAPD {
+  font-family: 'Crete Round', serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 32px;
+  line-height: 44px;
+  color: #fff;
+  padding-left: 15px;
+}
+
+.parrafo2 {
+  font-family: 'Inter', sans-serif;
+  padding-left: 15px;
+  padding-right: 30 px;
+  font-size: 16px;
+  color: #fff;
+}
+
+.list_programs2 {
+  display: flex;
+  align-content: space-between;
+  flex-direction: row;
+  padding: 10px 12px;
+}
+
+.boton2 {
+  width: 327px;
+  height: 48px;
+  background: #ff6b00;
+  font-family: 'Inter', sans-serif;
+  font-size: 17px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  letter-spacing: 0.03em;
+  color: #fff;
+}
+
+.page3 {
+  background-color: var(--v2);
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  margin-top: 0%;
+  justify-content: center;
+  background-image: url(./images/about.svg);
+  background-repeat: no-repeat;
+  background-position-y: bottom;
+  background-position-x: right;
+  padding-bottom: 100px;
+}
+
+.about {
+  font-family: 'Crete Round', serif;
+  padding-left: 15px;
+  padding-top: 40px;
+  line-height: 88px;
+  letter-spacing: 0.37px;
+  color: #ff6b00;
+  font-size: 72px;
+  font-weight: 400;
+}
+
+.parrafoAM {
+  font-family: 'Inter', sans-serif;
+  padding-left: 15px;
+  padding-right: 30 px;
+  font-size: 16px;
+  color: #fff;
+}
+
+.boton3 {
+  width: 161px;
+  height: 48px;
+  background: #ff6b00;
+  font-family: 'Inter', sans-serif;
+  font-size: 17px;
+  line-height: 24px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  color: #fff;
+  letter-spacing: 0.03em;
+}
+
+.vector2 {
+  color: #fff;
+  color: var(--v2);
+  left: 24px;
+  right: 24px;
+}
+
+.skills {
+  text-align: center;
+  color: #fff;
+}
+
+.bigskill {
+  font-family: 'Inter', sans-serif;
+  font-size: 32px;
+  list-style-type: none;
+  font-weight: 700;
+  line-height: 44px;
+}
+
+.smallskill {
+  font-family: 'Inter', sans-serif;
+  font-size: 22px;
+  list-style-type: none;
+  font-weight: 400;
+  line-height: 43px;
+}
+
+.pfinal {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 44px;
+  text-align: center;
+  color: #172b4d;
+  padding-left: 15px;
+  padding-right: 30px;
+}
+
+.fill {
+  font-family: 'Inter', sans-serif;
```

Comment:
- [ ] Kindly use the DRY principle of CSS rules, don't repeat a property so many times; instead, create a class or id and put the properties there when you need it; you call it. For further details, [Best practice HTML and CSS](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md)

Example 
```
.font{
  font-family: 'Inter', sans-serif;
}

```